experiments/tf_trainer/common/text_preprocessor.py

```diff
@@ -172,8 +186,8 @@ def new_model_fn(features, labels, mode, params, config):
       text_feature = features[text_feature_name]
       # Make sure all examples are length 300
       # TODO: Parameterize 300
-      text_feature = tf.pad(text_feature, [[0, 0], [0, 300]])
-      text_feature = text_feature[:, 0:300]
+      # text_feature = tf.pad(text_feature, [[0, 0], [0, 300]])
```

Comment:
Let's remove these commented lines. They will still exist in the revision history if needed.

Reply:
Missed that! Thanks and done!